lib/seed.ts

```diff
@@ -3,9 +3,223 @@ console.log("Seed file running...");
+        url: faker.internet.url(),
+      },
+    ],
+    creator: user._id,
+  };
+  //add to post based on type
+  switch (type) {
+    case NoteTypeEnum.Component:
+      post = {
+        ...post,
+        code: {
+          code: "import cmd from 'search';",
+          codePreviewImage: faker.image.urlLoremFlickr({ category: "code" }),
+        },
+      };
+      break;
+    case NoteTypeEnum.Knowledge:
+      post = {
+        ...post,
+        whatYouLearned: Array.from({ length: faker.number.int(5) }, () =>
+          faker.hacker.phrase()
+        ),
+      };
+      break;
+    case NoteTypeEnum.Workflow:
+      post = {
+        ...post,
+        stepsToFollow: Array.from({ length: faker.number.int(5) }, () =>
+          faker.hacker.phrase()
+        ),
+      };
+      break;
+    default:
+      return;
+  }
+
+  try {
+    const newNote = await Note.create({
+      ...post,
+    });
+
+    await User.findOneAndUpdate(
+      { _id: user.id },
+      { $push: { notes: newNote.id } }
+    );
+  } catch (e) {
+    console.log(e);
+  }
+}
+
+async function makeExampleUser() {
+  const name = {
+    firstName: "Sirius",
+    lastName: "Black",
+  };
+  const email = "sirius.black@hogwarts.edu";
+  const uniqueUsername = generateFromEmail(email ?? "", 4);
+  const user = await User.create({
+    email: email,
+    password: "123123",
+    name: name.firstName + " " + name.lastName,
+    username: uniqueUsername,
+  });
+
+  await onboardAUser(
+    user,
+    "https://utfs.io/f/bbedbcdd-64ed-48b3-aa7b-8d7162545bc1-mg50ir.jpeg"
+  );
+
+  console.log("Example user created.");
+
+  console.log("Making posts for the example user...");
+  for (let i = 0; i < 5; i++) {
+    await makeNewPost(user);
+  }
+}
 
 async function seed() {
   await connectToDB();
+  await User.deleteMany({});
+  await Note.deleteMany({});
+  console.log("All users and notes cleared");
+  await makeRandomUsers(5);
+  console.log("5 random users created");
+  await makeExampleUser();
+  console.log("Example user created");
+  console.log("Seed completed");
 }
 
 seed();
+
+// {
+//   type: NoteType.Component,
+//   title: "Search Command",
+//   content:
+//     "### Dic quaeque gaudent; \n[illas ambo](http://simulatoremque.org/), virgineos fulgebant vices et frondes [mortisneu](http://www.achaide.net/voluntas-virtute.php) tribus recipit: conloquiumquemagnaeque. Tamen illa pectore!",
+//   description:
+//     "A mobile navigation that seamlessly reveals a full-height sheet designed to effortlessly accommodate extensive menus.",
+//   tags: ["Search Command"],
+//   id: 8,
+//   createdAt: new Date("2024-05-01"),
+//   updatedAt: new Date("2024-05-01"),
+//   numberOfStars: 10,
+//   numberOfViews: 100,
+//   code: {
+//     code: "import cmd from 'search';",
+//     codePreviewImage: "",
+//   },
+//   resourcesAndLinks: [
+//     {
+//       resource: "This is a resource",
+//       url: "https://www.google.com",
+//     },
+//   ],
+// },
```

Comment:
We can delete this if it's no longer important.  Let's not leave comment graveyards around if they were just used as a reference.